cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -449,6 +453,12 @@ func main() {
 	}
 	defer counterAggregator.Close()
 
+	dropLabels := make(map[string]string)
```

Comment:
Use `map[string]bool` is more readable when you are using it as a hit-or-miss map.

A list of labels can be fine in this scenario assuming the number of labels to drop won't be large. If you sort the list the look-up can be faster.